src/components/DetailDropdown/DetailDropdown.tsx

```diff
@@ -5,32 +5,19 @@ import { useGSAP } from "@gsap/react";
 
 import { PlusMinusButton } from "../buttons/PlusMinusButton/PlusMinusButton";
 
-interface productDetailsProps {
+interface DetailsProps {
```

Comment:
Should export and import this interface to faq/data.ts instead of remaking. Also don't think it should have Props in the name, since it's not props